### ISSUE TITLE
Support for "param" documentation tag for delegates is incomplete in Intellisense

### ISSUE THREAD

**Issue by jhinder** [dotnet/roslyn#26713]
**Version Used**: VS 15.7.0

**Steps to Reproduce**:

1. Paste the code below into a C# file.
2. Open autocompletion at the first marked location.
3. Write `<param name="parameter">Parameter documentation</param>` at that location.
4. Open Signature Help at the second location.

```c#
/// <summary>Summary</summary>
/// $$
delegate T SomeDelegate<T>(object parameter);

void M(SomeDelegate<int> theDelegate)
{
    theDelegate($$);
}
```

**Expected Behavior**:

1. For the first location: `param name="parameter"` should be suggested in the list.
2. For the second location: if the parameter is documented, the documentation is shown in the Signature Help popup.

Basically, the behaviour should be identical to that of method parameters.

**Actual Behavior**:

1. For the first location: `param name=""` autocompletions are never suggested.
2. For the second location: if parameter documentation is given, it does not show up in Signature Help.

Some notes:

* Documentation for type parameters is fully supported at both locations.
* If you remove all documentation for the delegate and then type `///` (to auto-generate documentation), the `param` tag is generated.
* The parameter documentation is shown in the Object Browser window.

**Comment by sharwell** [comment]
> Documentation for type parameters is fully supported at both locations.

@jhinder This is a good bug report, but I want to thank you particularly for including this and the other notes. 💯
